src/library/scala/Array.scala

```diff
@@ -102,7 +101,7 @@ object Array extends FallbackArrayBuilding {
   def copy(src: AnyRef, srcPos: Int, dest: AnyRef, destPos: Int, length: Int) {
     val srcClass = src.getClass
     if (srcClass.isArray && dest.getClass.isAssignableFrom(srcClass))
-      arraycopy(src, srcPos, dest, destPos, length)
+      java.lang.System.arraycopy(src, srcPos, dest, destPos, length)
```

Comment:
Note: this probably isn't affected anyway, as the array that gets here is already `null`-ed out.
But at least  we're consistent :) (in which case `Platform.arraycopy` should probably be removed).
Also note, that in other cases (e.g. https://github.com/scala/scala/blob/40f7fce0af1da614d99048b024e1ff579635f0f2/src/compiler/scala/tools/nsc/backend/jvm/BCodeHelpers.scala#L461-L461) the `System` reference is not fully qualified.